overrides/scripts/Electronics.zs

```diff
@@ -745,7 +745,7 @@ assembler.findRecipe(7680, [<ore:circuitElite>.firstItem * 4, <gregtech:meta_ite
 //Item conduit - by hand
 recipes.addShaped(<enderio:item_item_conduit> * 4, [

```

Comment:
This recipe could benefit from the string-based generator since it's repetitive.

Reply:
In that case it may be worth doing for all the other conduits as well.

Reply:
I mention it specifically because it's changed in the commit, but yes eventually it would be good to refactor a lot of this stuff. Outside of the scope of the current PR, however.